### PR TITLE
Update http-endpoint.asciidoc

### PR DESCRIPTION
**Pull request by amrmotan**
## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: libbeat/docs/http-endpoint.asciidoc [hunk 1]
```diff
@@ -56,7 +56,7 @@ curl -XGET 'localhost:5066/?pretty'
 ["source","js",subs="attributes"]
 ----
 {
-  "beat": "metricbeat",
+  "beat": "heartbeat",
```
Comment (jsoriano) [inline libbeat/docs/http-endpoint.asciidoc:59]: Umm, I think this should be `{beatname_lc}`, so this is replaced by the lowercased name of the actual beat on each guide.
```suggestion
  "beat": "{beatname_lc}",
```

**Review by jsoriano — CHANGES_REQUESTED** [review]
Good catch! But the change should be applied in master first, could you please rebase the PR on master? Thanks!

**Comment by jsoriano** [comment]
You will also need to [sign the CLA](https://www.elastic.co/contributor-agreement), please.

**Comment by bmorelli25** [comment]
Replaced by https://github.com/elastic/beats/pull/27117.

**Pull request closed without merging**